docs/batch-changes/create-a-batch-change.mdx

```diff
@@ -4,14 +4,91 @@
 
 Batch Changes are created by writing a [batch spec](/batch-changes/batch-spec-yaml-reference) and executing that batch spec with the [Sourcegraph CLI](https://github.com/sourcegraph/src-cli) `src`.
 
+There are two ways of creating a batch change:
+
+1. On your Sourcegraph instance, with [server-side execution](#on-your-sourcegraph-instance)
+2. On your local machine, with the [Sourcegraph CLI](#using-the-sourcegraph-cli)
+
+## On your Sourcegraph instance
+
+Here, you'll learn how to create and run a batch change via server-side execution.
+
+To get started, click the **Batch Changes** icon in the top navigation or navigate to `/batch-changes`.
+
+### Create a batch change
+
+Click the **Create batch change** button on the Batch Changes page, or go to `/batch-changes/create`.
+
+![create_batch_change_button](https://sourcegraphstatic.com/docs/images/batch_changes/2025/create_batch_change_button.png)
+
+You will be redirected to a page showing you a list of curated templates.
+
+### Choosing a template
+
+<Callout type="info">Templates is a feature available in Sourcegraph 6.6 and later.</Callout>
```

Comment:
If we replace "more" with "higher" earlier on, just make sure to replace this "later" with "higher" too (or just replace all the instances of "more" with "later" to be consistent)